src/components/RefinementList/RefinementList.js

```diff
@@ -80,36 +81,36 @@ class RefinementList extends React.Component {
   //
   // Finally, we always stop propagation of the event to avoid multiple levels RefinementLists to fail: click
   // on child would click on parent also
-  handleItemClick(value, e) {
-    if (isSpecialClick(e)) {
+  handleItemClick({facetValueName, originalEvent, isRefined}) {
```

Comment:
I think it is very convenient for user facing API, but it still add an overhead (even small). Your call :)